sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -78,6 +79,26 @@ final class DataFrameWriter private[sql](df: DataFrame) {
   }
 
   /**
+   * Set the trigger interval for the stream query.
+   *
+   * @since 2.0.0
+   */
```

Comment:
We should remove `@experimental` for all the dataframe stuff and mark just the streaming methods.

/cc @rxin